### ISSUE TITLE
Ark server container failing with: "missing location in azure configuration"

### ISSUE THREAD

**Issue by ffd2subroutine** [heptio/ark#397]
Ark server pod in "Waiting:CrashLoopBackOff" state

```
time="2018-03-22T13:03:27Z" level=info msg="setting log-level to INFO"
time="2018-03-22T13:03:27Z" level=info msg="Starting Ark server v0.7.1" logSource="pkg/cmd/server/server.go:88"
time="2018-03-22T13:03:27Z" level=info msg="Ensuring namespace exists for backups" logSource="pkg/cmd/server/server.go:253" namespace=heptio-ark
time="2018-03-22T13:03:27Z" level=info msg="Namespace already exists" logSource="pkg/cmd/server/server.go:265" namespace=heptio-ark
time="2018-03-22T13:03:27Z" level=info msg="Retrieving Ark configuration" logSource="pkg/cmd/server/server.go:270"
time="2018-03-22T13:03:27Z" level=info msg="Successfully retrieved Ark configuration" logSource="pkg/cmd/server/server.go:288"
time="2018-03-22T13:03:27Z" level=info msg="Using default resource priorities" logSource="pkg/cmd/server/server.go:322" priorities="[namespaces persistentvolumes persistentvolumeclaims secrets configmaps serviceaccounts]"
time="2018-03-22T13:03:27Z" level=info msg="Configuring cloud provider for backup service" logSource="pkg/cmd/server/server.go:362"
time="2018-03-22T13:03:27Z" level=info msg="Configuring cloud provider for snapshot service" logSource="pkg/cmd/server/server.go:378"
An error occurred: rpc error: code = Unknown desc = missing location in azure configuration 
```

**Comment by ncdc** [comment]
Hi @ffd2subroutine, it looks like you're using a combination of our latest development version from the master branch with our latest published release, v0.7.1.

Please make sure you check out the v0.7.1 tag and follow the instructions there. We are working on updating our documentation to make this clearer.

Tag: https://github.com/heptio/ark/tree/v0.7.1

Docs for v0.7.1: https://heptio.github.io/ark/v0.7.1/

Please give this a try and let us know if it works.

**Comment by ffd2subroutine** [comment]
Hi @ncdc, already found it:), it works now, this can be closed.
Thank you.
